src/Utility/Hash.php

```diff
@@ -138,9 +138,6 @@ public static function extract($data, $path)
         }
 
         if (strpos($path, '[') === false) {
-            if (function_exists('array_column') && preg_match('|^\{n\}\.(\w+)$|', $path, $matches)) {
```

Comment:
@jrbasso As long as the tests pass I am fine with any optimizations you see fit :)